### PR TITLE
Ibisstore fix table columns & link to store

### PR DESCRIPTION
**Pull request by Matthbo**
Improves ibisstore table columns logic & adds configuration to the api response so table can correctly link to the right store (was undefined in frontend before)

### CODE REVIEW

**Comment by Matthbo** [comment]
Unit tests run in java 11 on jenkins, instead of java 16 which is when `record` class type was added.
I guess I should convert it back to a class to comply with java 11 code

**Comment by nielsm5** [comment]
> Unit tests run in java 11 on jenkins, instead of java 16 which is when `record` class type was added. I guess I should convert it back to a class to comply with java 11 code

Sounds like a Jenkins thing to me ;)

**Comment by Matthbo** [comment]
Thanks Sonar but I'm not rewriting the whole class for a simple addition

File: console/frontend/src/main/frontend/src/app/views/ibisstore-summary/ibisstore-summary.component.html [hunk 1]
```diff
@@ -79,20 +79,22 @@ <h4>Result</h4>
...
-                    [attr.rowspan]="t.slotcount"
-                    title="{{ t.name }}"
-                  >
-                    {{ t.type }}
-                  </td>
-                  <td *ngIf="s.first" [attr.rowspan]="t.slotcount">
-                    {{ t.msgcount }}
-                  </td>
+                <tr *ngFor="let s of t.slots; let first = first">
+                  <ng-container *ngIf="first; else notFirstSlot">
```
Comment (nielsm5) [inline console/frontend/src/main/frontend/src/app/views/ibisstore-summary/ibisstore-summary.component.html:83]: Voor de volgende keer 😄 
```suggestion
                <tr *ngFor="let s of t.slots; let firstElement = first">
                  <ng-container *ngIf="firstElement; else notFirstSlot">
```

**Review by nielsm5 — APPROVED** [review]

**Review by philipsens — APPROVED** [review]

**Pull request merged**